Dockerfile

```diff
@@ -11,6 +9,17 @@ RUN apk add gcc g++ make libffi-dev openssl-dev python3-dev build-base --update-
 RUN apk add nodejs git postgresql postgresql-dev
 RUN mkdir /ve
 RUN pyvenv --system-site-packages /ve/std
-RUN pip install -r /autoapi/requirements.txt
-RUN npm install
+
+COPY requirements.txt /
```

Comment:
Copying `requirements.txt` separately from the rest of the source code means that *only* changes to `requirements.txt` trigger a rebuild of the fs layer containing our python dependencies, saving both time and download bandwidth.